### DISCUSSION TITLE
Python "Cant open file.. Errno 2 No such file or directory

### DISCUSSION THREAD

**Discussion by bel876** [community/community#21502] [Other Feature Feedback, Questions, & Ideas]
<p>Hi. Im a newbie here. I dont have coding experience. I wanted to run faceit_live.py and i get this result “cant open file…”</p>
<p>Am trying to run the program faceit_live using the first order model which I have read in one of the forums here by alew3</p>
<p>in conda, i did type “conda activate faceit_live3”</p>
<p>but when i typed next "python faceit_live.py, it says "python: cant open ‘faceit_live.py’: [Errno 2] No such file or directory</p>
<p>What should I do? I would really appreciate the help. Thanks!</p>

**Comment by wordsoup** [comment]
<p>Hi <a class="mention" href="/bel876">@bel876</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the GitHub <a href="/t5/Programming-Help-and-Discussion/bd-p/programming">Programming Help and Discussion</a> board.</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Python is telling you that there is no copy of “faceit_live.py” in the directory you are in.</p>
<p>“faceit_live.py” is part of <a href="https://github.com/alew3/faceit_live3" rel="nofollow noopener">the repository</a> you needed to download or clone in order to work with it.</p>
<p>You might review <a href="https://github.com/alew3/faceit_live3/blob/master/README.md" rel="nofollow noopener">the readme file</a> to make sure you’ve followed the download and setup steps correctly. If you are still having problems you might get a better response by posting your question <a href="https://github.com/alew3/faceit_live3/issues" rel="nofollow noopener">as an issue</a> in the repository.</p>
